tools/release-tools/setVersion.js

```diff
@@ -462,7 +462,7 @@ const setPackageJsonRelease = () => {
                 }
 
                 if (theFilename.indexOf("/package.json") !== -1) {
-                    let srcFolder = theFilename.replace("/package.json", "/**/*.ts");
+                    let srcFolder = theFilename.replace("/package.json", "/**/*.ts", "/**/*.tsx", "/**/*.html");
```

Comment:
Just update our build level version replacement logic to support updating the build versions in *.tsx and *.html files  -- this is used for manually updating versions via `npm run setVersion -- 2.x.x` and the nightly builds